timbermill-java/timbermill-client/src/main/java/com/datorama/oss/timbermill/unit/Event.java

```diff
@@ -36,6 +34,7 @@
 
 	protected String taskId;
 	private static final Logger LOG = LoggerFactory.getLogger(Event.class);
+	protected static final long serialVersionUID = 20201201; // date
 
```

Comment:
I read on the web that serialVersionUID doesn't affect the inheriting classes,
so had to add this member to SpotEvent, StartEvent...